cmd/get/execution.go

```diff
@@ -0,0 +1,68 @@
+package get
+
+import (
+	"context"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flytestdlib/logger"
+
+	"github.com/lyft/flytectl/cmd/config"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+	"github.com/lyft/flytectl/pkg/printer"
+)
+
+var executionColumns = []printer.Column{
+	{"Name", "$.id.name"},
+	{"Workflow Name", "$.closure.workflowId.name"},
+	{"Type", "$.closure.workflowId.resourceType"},
+	{"Phase", "$.closure.phase"},
+	{"Started", "$.closure.startedAt"},
+	{"Elapsed Time", "$.closure.duration"},
+}
+
+func ExecutionToProtoMessages(l []*admin.Execution) []proto.Message {
+	messages := make([]proto.Message, 0, len(l))
+	for _, m := range l {
+		messages = append(messages, m)
+	}
+	return messages
+}
+
+func getExecutionFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
```

Comment:
I know the repo does not have many unit tests, but can you please start adding to the commands that you added? And I will ask all the contributors to add more tests.